### PR TITLE
[TASK-13] As a user, I can view survey lists in home screen

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/surveyor/projects/1#card-43745490

## What happened
View survey list in the home screen
- On the home screen, each survey card must display the following info:
   1. Cover image (background)
   2. Name (in bold)
   3. Description
   4. Horizontal scroll through the surveys.
 
## Insight
- [x] Implement list screen
- [x] Implement screen styles

Used [slick-react](https://github.com/akiran/react-slick) to show the list and horizontal list view. 

## Proof Of Work
image

### CODE REVIEW

File: src/screens/Home/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Header from 'screens/Home/header';
 
 const Home = () => {
   return (
-    <PageBackground>
+    <PageBackground type='default'>
```
Comment (namtx) [inline src/screens/Home/index.js]: we need a consistency of single and double qoute here. 
```suggestion
    <PageBackground type="default">
```

File: src/screens/Home/surveyList.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useRef } from 'react';
+
```
Comment (namtx) [inline src/screens/Home/surveyList.js:2]: minor thing 🚐  but the line break here is redundant IMO

File: src/screens/Home/surveyList.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const SurveyList = () => {
+  const sliderRef = useRef(null);
+
+  useEffect(() => {
+    window.addEventListener('wheel', (e) => {
+      slideList(e.wheelDelta);
+    })
+  }, []);
+
+  const slideList = (y) => {
```
Comment (namtx) [inline src/screens/Home/surveyList.js]: we need a more meaningful variable name here 🤔

File: src/screens/Home/surveyList.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    y > 0 ? (
+      sliderRef.current.slickNext()
+    ) : (
+      sliderRef.current.slickPrev()
+    )
+  }
+
+  const settings = {
```
Comment (namtx) [inline src/screens/Home/surveyList.js]: do we need to move this variable `settings` out of the component?

File: src/stylesheets/vendor/slick/slick.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+@import '~slick-carousel/slick/slick.css';
+@import '~slick-carousel/slick/slick-theme.css';
+
```
Comment (namtx) [inline src/stylesheets/vendor/slick/slick.scss]: minor thing but this line could be removed

File: src/tests/mocks/matchMedia.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+window.matchMedia = jest.fn().mockImplementation(query => {
```
Comment (namtx) [inline src/tests/mocks/matchMedia.js]: for the function body just only have `return` statement, I think we can ommit the `return`
```suggestion
window.matchMedia = jest.fn().mockImplementation(query => ({
matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(),
    removeListener: jest.fn(),
    }))
```

File: src/screens/Home/surveyList.js (hunk 2 above)
Comment (rafayet-monon) [reply src/screens/Home/surveyList.js:2]: @namtx If you remember we used `import/order` of `eslint` to add the line breaks - https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m

**Comment by rafayet-monon** [comment]
@namtx Fixed most of the reviews in - [c5efbc8](https://github.com/rafayet-monon/surveyor/pull/16/commits/c5efbc8bf6a6d67c0417854dd8aae7f2a5e97c6a) 

Commented in one - https://github.com/rafayet-monon/surveyor/pull/16#discussion_r508977426

**Review by namtx — APPROVED** [review]

File: src/screens/Home/surveyList.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      sliderRef.current.slickNext()
+    ) : (
+      sliderRef.current.slickPrev()
+    )
+  }
+
+  return (
+    <div className="survey-list">
+      <Slider { ...slickSettings } ref={ sliderRef }>
+        <div>
```
Comment (Lahphim) [inline src/screens/Home/surveyList.js:28]: @rafayet-monon It seems fixed content 🤔
Should we make it more flexible content?

File: src/screens/Home/surveyList.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        </div>
+      </Slider>
+
+      <h3 className="survey-list__title">Working from home Check-In</h3>
+      <p className="survey-list__description">
+        We would like to know about how you feel about our work from home.
+      </p>
+
+      <div className="survey-list__navigation">
+        <button className="button button--circle button--circle-right">
```
Comment (Lahphim) [inline src/screens/Home/surveyList.js]: @rafayet-monon what do you think if we design a structure like this?
```html
<div>
  <div>
    <h3 />
    <p />
  </div>

  <div>
    <button />
  </div>
</div>
```
Then use flex to layout them.
So we can omit this `button--circle-right`
Something like this https://jsfiddle.net/lahphim/bsfpw7L0/

Comment (rafayet-monon) [reply src/screens/Home/surveyList.js]: @Lahphim fixed in [9b8b432](https://github.com/rafayet-monon/surveyor/pull/16/commits/9b8b432e092af9f7a330f774af2434887167c89b).

**Review by Lahphim — APPROVED** [review]

**Pull request merged**